.github/workflows/build.yml

```diff
@@ -298,3 +298,73 @@ jobs:
         if-no-files-found: error
         retention-days: 30
 
+  archlinux:
+    strategy:
+      matrix:
+        arch:
+          [
+            "linux/amd64 x86_64",
+            "linux/arm64/v8 aarch64",
```

Comment:
I really like that you are testing more than one architecture. I should do the same for the other OSes too :)